egs/librispeech/ASR/zapformer/train.py

```
@@ -0,0 +1,1796 @@
+try:
+    from batched_rubik import BatchedRubik
+    # could also have done:
+    # from rubik import Rubik
+except:
+    pass
+
+
+from variable_combined_scheduler import VariableCombinedLRScheduler
+try:
+    from variable_combined_scheduler import InterpCosineLRScheduler
+    LRSchedulerType = VariableCombinedLRSchedule
```

Ruff (0.15.12)

[error] Undefined name `VariableCombinedLRSchedule`

(F821)